release.sh

```diff
@@ -11,4 +11,8 @@ node tasks/update-versions
 #node tasks/publish-search --engine ember-guides --environment production --revision $VERSION --api-key $API_KEY
 git add --all
 git commit -m "Add snapshot for Ember.js revision $VERSION"
+
+# Staging has shifted to the newer Firebase platform
+firebase deploy --project staging
```

Comment:
@locks FYI.  Gets to be a bit of a nuisance atm as we need the older Firebase CLI to deploy to production and the newer CLI to deploy to staging ...